backend/fms_core/tests/constants.py

```diff
@@ -46,7 +46,7 @@ def create_sample(sample_kind, individual, container, coordinates='', **kwargs):
     }
 
 
-def create_extracted_sample(sample_kind, individual, container, volume_used, coordinates='', **kwargs):
+def create_extracted_sample(sample_kind, individual, container, volume, coordinates='', **kwargs):
```

Comment:
I'm not sure i get all those changes here ... Removing volume_used is fine but setting the volume for all the following test to volume_used is likely to break those test since this is not the intended behaviour (volume should not be = volume_used but to the value stored in the template for volume. the parent volume should = volume - template_volume_used. You can put a param to set volume but set it to 0 by default as it is the current behaviour.